README.md

```diff
@@ -187,28 +188,29 @@ To use the Square API, you import the Client class, instantiate a Client object,
 
 ```ruby
 client = Square::Client.new(
-    access_token: 'SANDBOX ACCESS TOKEN HERE',
-    environment: 'sandbox'
+  access_token: 'SANDBOX ACCESS TOKEN HERE',
+  environment: 'sandbox'
 )
 ```
 
 - To access production resources, set environment to production:
 
 ```ruby
 client = Square::Client.new(
-    access_token: 'ACCESS TOKEN HERE',
-    environment: 'production'
+  access_token: 'ACCESS TOKEN HERE',
+  environment: 'production'
 )
 ```
 
 - To set a custom environment provide a `custom_url`, and set environment to `custom`:
 
 ```ruby
 client = Square::Client.new(
-    access_token:'ACCESS TOKEN HERE',
-    environment: 'custom',
-    custom_url: 'https://your.customdomain.com'
+  access_token:'ACCESS TOKEN HERE',
+  environment: 'custom',
+  custom_url: 'https://your.customdomain.com'
 )
+```
```

Comment:
This one is missing an end-code-block which causes other parts of the document to render like code.